contracts/drafts/ERC777/ERC777.sol

```diff
@@ -25,8 +25,13 @@ contract ERC777 is IERC777, IERC20 {
     string private _name;
     string private _symbol;
 
-    bytes32 constant private TOKENS_SENDER_INTERFACE_HASH = keccak256("ERC777TokensSender");
-    bytes32 constant private TOKENS_RECIPIENT_INTERFACE_HASH = keccak256("ERC777TokensRecipient");
+    // We inline the result of these hashes because Solidity does not calculate them in compile time.
+    // See https://github.com/ethereum/solidity/issues/4024.
+    //
+    // keccak256("TOKENS_SENDER_INTERFACE_HASH")
+    bytes32 constant private TOKENS_SENDER_INTERFACE_HASH = hex"29ddb589b1fb5fc7cf394961c1adf5f8c6454761adf795e67fe149f658abe895";
```

Comment:
Should we switch to this new style for the old constants too?

Reply:
No, I just forgot about that other style. I'm switching this one to the other one because it's smarter about complaining when there are missing digits.